receiver/jmxreceiver/config.go

```diff
@@ -59,6 +59,9 @@ type Config struct {
 	RemoteProfile string `mapstructure:"remote_profile"`
 	// The SASL/DIGEST-MD5 realm
 	Realm string `mapstructure:"realm"`
+	// Array of property=value strings to pass as system properties when running JMX Metric Gatherer
+	Properties       []string `mapstructure:"properties"`
```

Comment:
Any reason not to use a `map[string]string` here instead? That way you don't have to parse the configuration yourself

Reply:
viper is destructive in its case insensitivity and this avoid that issue: https://github.com/open-telemetry/opentelemetry-collector/issues/2594

Reply:
I see. I don't have enough context to judge here but it feels like the correct solution would be to fix the issue above then instead of working around it (we don't expose Viper explicitly anymore so the issue can be addressed more easily).

Reply:
That sounds fair and I've updated to a map and will try to address the underlying issue in the core project.